sbom/main.ts

```diff
@@ -11,12 +11,16 @@ interface Dependency {
     version: string;
 }
 
+interface DependencyList {
+    [name: string]: string;
```

Comment:
This looks good and checks out in the TypeScript playground: https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgCIQA4RAE2wgTwBlgBnMZAbwFgAoZB5AbRDgFsIAuZcqUAcwC63XgIDcdAL506oSLEQoACogDWcfihr1GrDiLB8Q-CToYA3aKWAB7EAaMm6jZABtgSEKS49D454xgBFgO-mbIeFi4+MAQpNzoUXgghCTkptK0dK4QFBjcKgjqmsgAvFQBDABEehBV3FUYahp1ADSVyFWWUNZ29Z0ADAB0wwCMVe3hVe6e3v1VALIAkgAqEx1VQVjzIBhs61OR2MkIsaT92i4uVUfRKQQA+k1FLQ-jDWMjB1eMN5jH+Eez2KEAeACZ5sMwV9Ji5Mpk6Ag7KQbDkhq4bPwABQAKQAygB5AByQ1ExmAMAIWIwrWQAFdojBQBAcLSwQBKdliIA

https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures uses `[index: string]` and I remembered using `[key: string]` back in the day in TypeScript work.

https://basarat.gitbook.io/typescript/type-system/index-signatures clears that up:

> TIP: the name of the index signature e.g. index in { [index:string] : {message: string} } has no significance for TypeScript and is only for readability. e.g. if it's user names you can do { [username:string] : {message: string} } to help the next dev who looks at the code (which just might happen to be you).